tls-tutorial/doc/use_your_own_certificate.md

```diff
@@ -0,0 +1,685 @@
+
+Exasol's [docker-db](https://github.com/exasol/docker-db) provides a Docker container that is quite convenient for experiments like our tutorial.
+
+The following docker command downloads the container's layers and afterwards starts the container. Expect a couple of minutes until the download and database start are done.
+
+```shell
+docker run --name exasoldb -p 8563:8563 -p 2580:2580 --detach --privileged --stop-timeout 120  exasol/docker-db:7.1.17
+```
+
+There are two ports forwarded to the host:
+
+- 8563: database port
+- 2580: [BucketFS] port
+
+### Installing the MySQL JDBC driver in Exasol
+
+[Import](https://docs.exasol.com/db/latest/sql/import.htm) into Exasol requires that the database driver of the source database (MySQl in our case) is installed in Exasol.
+
+Installation in the Docker variant requires uploading the driver to a [BucketFS] bucket and updating a configuration file.
+
+1. Download the [MySQL Connector] (JDBC driver)
+   ```shell
+   wget https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-j-8.0.31.tar.gz
+   ```
+2. Get the ID of the Exasol Docker container
+   ```shell
+   container_id=$(docker ps -a | grep exasol | sed -e's/ .*//')
+   ```
+3. Get the write-password of the default Bucket in BucketFS
+   ```shell
+   write_pwd=$(docker exec -it "$container_id" cat /exa/etc/EXAConf | grep WritePasswd | sed -e's/^.* = //' -e's/[\n\r]//g' | base64 --decode)
+   ```
+4. Copy the driver to the default Bucket
+   ```shell
+   curl -vX PUT -T mysql-connector-j-8.0.31/mysql-connector-j-8.0.31.jar "http://w:$write_pwd@localhost:2580/default/drivers/jdbc/mysql-connector-j-8.0.31.jar"
+   ```
+5. Create a driver configuration (`settings.cfg`)
+   ```shell
+   echo 'DRIVERNAME=MYSQL_JDBC
+   JAR=mysql-connector-j-8.0.31.jar
+   DRIVERMAIN=com.mysql.cj.jdbc.Driver
+   PREFIX=jdbc:mysql:
+   NOSECURITY=YES
+   FETCHSIZE=100000
+   INSERTSIZE=-1' > settings.cfg
+   ```
+6. Upload the configuration to `drivers/jdbc` in the default bucket:
+   ```shell
+   curl -vX PUT -T settings.cfg "http://w:$write_pwd@localhost:2580/default/drivers/jdbc/settings.cfg"
+   ```
+7. Verify this installation:
+   ```shell
+   curl http://localhost:2580/default
+   ```
+   Must produce a listing that looks similar to this:
+   ```
+   EXAClusterOS/ScriptLanguages-standard-EXASOL-7.1.0-slc-v4.0.0-CM4RWW6R.tar.gz
+   drivers/jdbc/mysql-connector-j-8.0.31.jar
+   drivers/jdbc/settings.cfg
+   ```
+   
+### Copying the Certificate to Exasol
+
+**Information:** this part will be added in a separate pull request.
+
+## Run the Import
+
+In this last part of the tutorial, we import the data from the `shapes` database and `shapes` table in MySQl into Exasol.
+
+One last piece of preparation is that you get the IP address of the Docker host (i.e. the Ubuntu machine you run the tutorial on).
+
+```shell
+ip address show docker0 | grep inet
+```
+
+Please, note down the IP address. You will need it for the `IMPORT` command.
+
```

Comment:
Isn’t that always 172.17.0.1 ?

Reply:
Unfortunately not. It depends on the setup of the Docker host.